processes/frbr/classify.py

```diff
@@ -14,110 +14,110 @@
 
 
 class ClassifyProcess(CoreProcess):
+    WINDOW_SIZE = 100
+
     def __init__(self, *args):
         super(ClassifyProcess, self).__init__(*args[:4], batchSize=50)
 
-        self.ingestLimit = int(args[4]) if args[4] else None
+        self.ingestLimit = int(args[4]) if len(args) >= 5 and args[4] else None
```

Comment:
ingestLimit needs to stay the same because its used by the CoreProcess windowQuery function